### ISSUE TITLE
Avalonia Sample

### ISSUE THREAD

**Issue by StarlkYT** [CommunityToolkit/MVVM-Samples#58]
Following from my comment in #8.
I'd like to create a sample project for Avalonia.

**Comment by michael-hawker** [comment]
Thanks @StarlkYT, our goal is to have similar looking sample apps showcasing the MVVM Toolkit across the different platforms all using the shared MVVM core. @Sergio0694 can help point you to any specifics about the samples as he wrote the original app.

Let us know if you need a hand.

We had also discussed having another sample folder for smaller more independent samples that stand on their own, so depending on the scope you wanted to take on, we could figure out a plan for that too.

**Comment by StarlkYT** [comment]
> Thanks @StarlkYT, our goal is to have similar looking sample apps showcasing the MVVM Toolkit across the different platforms all using the shared MVVM core. @Sergio0694 can help point you to any specifics about the samples as he wrote the original app.
> 
> Let us know if you need a hand.
> 
> We had also discussed having another sample folder for smaller more independent samples that stand on their own, so depending on the scope you wanted to take on, we could figure out a plan for that too.

Cool! I will try my best to make the app look similar to the other samples.

**Comment by pwm1234-sri** [comment]
@StarlkYT @michael-hawker Any news on a demo project using the MVVM Toolkit with Avalonia? I am trying to learn both the MVVM Toolkit and Avalonia. Let me know if you have anything or suggestions on how to get started. Thanks!

**Comment by StarlkYT** [comment]
> @StarlkYT @michael-hawker Any news on a demo project using the MVVM Toolkit with Avalonia? I am trying to learn both the MVVM Toolkit and Avalonia. Let me know if you have anything or suggestions on how to get started. Thanks!

Hi, Avalonia works just fine with the Community.Toolkit, in fact, there's a template that comes with CT set up.
I didn't have much of free time to work on the samples.

**Comment by pwm1234-sri** [comment]
Yes, I discovered the Avalonia template that uses the CT for MVVM. Thanks! I also found a few excellent articles by @bsonnino; there is a repo most relevant to this topic at [MvvmAvalonia](https://github.com/bsonnino/MvvmAvalonia).

But it would be great if there was an Avalonia implementation of this MVVM-Samples application.
